test/quicklink.spec.js

```diff
@@ -1,344 +1,358 @@
-const sleep = ms => new Promise(r => setTimeout(r, ms));
+const assert = require('assert').strict;
```

Comment:
Could also use `uvu/assert` in case you didn’t know :)

Reply:
I read about it, but haven't looked closer :)

I'll have a look again, thanks!

Reply:
BTW there's a failing test since the beginning when you added it. Any chance you could have a look?

https://github.com/GoogleChromeLabs/quicklink/blob/7d49abcec4de2b781e3435b3c62a3a9950fa5237/test/quicklink.spec.js#L232